apps/meteor/app/models/server/models/Subscriptions.js

```diff
@@ -28,9 +28,9 @@ export class Subscriptions extends Base {
 		this.tryEnsureIndex({ desktopNotifications: 1 }, { sparse: 1 });
 		this.tryEnsureIndex({ mobilePushNotifications: 1 }, { sparse: 1 });
 		this.tryEnsureIndex({ emailNotifications: 1 }, { sparse: 1 });
+		this.tryEnsureIndex({ audioNotifications: 1 }, { sparse: 1 });
```

Comment:
not sure where this came from 🤔

Reply:
Hmmm, copied this from the old PR. I kinda remember this prop was used on "notifications", but cannot find any ref to it anymore